src/pages/resultsView/coExpression/CoExpressionTab.tsx

```diff
@@ -355,6 +356,9 @@ export default class CoExpressionTab extends React.Component<
                         this.props.store.entrezGeneIdToReferenceGene.result ||
                         {};
 
+                    const entrezGeneIdToGene =
```

Comment:
entrezGeneIdToGeneMap?

Reply:
Yeah, makes more sense. We already have another variable called `entrezGeneIdToReferenceGene`. We should probably rename that one as well.